language/orchestration/langchain/Retrieval Augmented Generation - Contract Analyst.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    from langchain.vectorstores import Chroma
</pre><p>Put all imports at the top of the notebook as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>